tests/gem/packet_test_data.py

```diff
@@ -779,6 +779,17 @@
         "time_stamp": datetime.datetime(2018, 6, 11, 21, 16, 58),
         "voltage": 122.2,
     },
+    "ECM-1240.bin": {
```

Comment:
Likewise, should this this be under something like `tests/ecm/packet_test_data.py`?